styleguide/derek/view-templates/blog/article.ejs

```diff
@@ -0,0 +1,75 @@
+<div class="standard-padding">
+  <div class="container">
+    <div class="col-md-8">
+      <h1>Report: Enterprises Turn to Managed Services to Optimize Public Cloud</h1>
+      <p>By <span class="rsBlog-author-name">Mike Rasterize De Yellow</span> -April 24, 2019</p>
+      <p class="center">
+        <img src="../imgs/derek/alpaca.jpg" alt="title here" />
+      </p>
+      <p>
+        Texty McText Dreamcatcher hashtag organic bushwick lumbersexual hot chicken health goth venmo. Tbh lo-fi sriracha butcher put a bird on it neutra activated charcoal readymade hashtag farm-to-table. Sustainable XOXO succulents pickled tilde leggings iceland church-key. Gluten-free readymade activated charcoal locavore, cornhole iceland tote bag fashion axe tumeric. Pork belly gastropub vexillologist, vinyl af asymmetrical before they sold out venmo ramps letterpress migas. Woke 8-bit synth whatever, fanny pack viral palo santo cray everyday carry.
+      </p>
+      <p>
+        Gochujang disrupt bicycle rights everyday carry bitters blog cornhole portland fanny pack sartorial man braid heirloom kitsch. Roof party banh mi disrupt jean shorts williamsburg. Poutine vexillologist taiyaki, crucifix woke portland fingerstache farm-to-table street art 3 wolf moon. Chia fam migas +1, vinyl air plant craft beer tumeric activated charcoal etsy.
+      </p>
+      <p>
+        Narwhal vape banjo readymade lumbersexual next level put a bird on it, banh mi pinterest YOLO copper mug kogi gochujang jianbing bushwick. Normcore lomo gastropub schlitz photo booth thundercats. Biodiesel literally single-origin coffee, meggings sartorial edison bulb lo-fi keffiyeh williamsburg vape. Coloring book sriracha literally microdosing, aesthetic typewriter austin. Deep v fanny pack chartreuse, williamsburg sustainable iceland hexagon lumbersexual cliche sriracha umami authentic ramps freegan celiac. Cold-pressed adaptogen godard chillwave snackwave, celiac keytar vaporware tumblr jianbing YOLO cardigan shoreditch live-edge.
+      </p>
+      <p>
+        *For additional information on Rackspace, please visit <a href="#">ww.rackspace.com</a>
```

Comment:
I like the missing `w` on the subdomain here. It feels like a real blog article.